src/color-chart/color-chart.js

```diff
@@ -232,6 +258,52 @@ const Self = class ColorChart extends ColorElement {
 			// rawProp: "coord",
 		},
 
+		yRange: {
+			get () {
+				return this.yResolved.range ?? this.yResolved.refRange ?? [0, 100];
```

Comment:
Shouldn't this return the actual range if ymin/ymax are used?

Reply:
We use it to calculate `yMin` and `yMax` when set to `coord`. So, it's `yMin`/`yMax`, depending on range, not vice versa.

Reply:
Then I'm not sure we need this prop? It seems like it would be confusing for users, and it's not like it caches some particularly complex calculation.